photon-core/src/main/java/org/photonvision/vision/pipeline/Calibrate3dPipeline.java

```diff
@@ -69,15 +69,18 @@ public class Calibrate3dPipeline
 
     private static final FrameThresholdType PROCESSING_TYPE = FrameThresholdType.NONE;
 
-    public Calibrate3dPipeline(String uniqueName) {
-        this(12, uniqueName);
+    private final String uniqueCameraName;
+
+    public Calibrate3dPipeline(String uniqueCameraName) {
```

Comment:
Architecture wise idk if having Calibrate3dPipeline know about the camera unique name makes sense. can we instead move this logic into `tryCalibration` and pass the unqiue name in from there?

Reply:
yeah just instead of making it a parameter in the constructor, pass it in when needed is all

Reply:
I would argue that this method is much more messy.

Reply:
The general rule is that pipes shouldn't know anything about cameras. this can be broken/bent for calibration but in general that's my rule of thumb